src/main/java/seedu/address/model/module/AcademicYear.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.model.module;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Module's academic year in the address book.
+ * Guarantees: immutable, is valid as declared in {@link #isValidYear(Integer)} (Integer)}
+ *
+ * @author waytan
+ */
+public class AcademicYear {
+    public static final String MESSAGE_ACADEMICYEAR_CONSTRAINTS =
+            "Academic Year should be a 4 digit number, with the first and last 2 digits representing the calendar year.";
+
+    public final Integer yearNumber;
```

Comment:
Is there a reason this attribute is public?

Reply:
It's public in order to be consistent with the standards used in Person and associated classes (such as Name, Address, Email).